src/client/pages/_slug.vue

```diff
@@ -62,39 +31,21 @@ export default {
         socialTarget: this.url
       })
     }
-  }
-}
-</script>
-
-<style>
-@import '../components/app-core/index.css';
-
-.page__content {
-  margin: 0 auto;
-  padding: 0 var(--spacing-default);
-  max-width: 920px;
-}
+  },
 
-.page__sidebar {
-  margin: 0 auto var(--spacing-double) auto;
-  width: 100%;
-  max-width: var(--max-width);
-}
+  computed: {
+    shareText() { return this.page.seo.description },
+    shareTitle() { return this.page.seo.title },
+  },
 
-@media screen and (min-width: 880px) {
-  .page__content {
-    display: flex;
-    justify-content: space-between;
-    align-items: flex-start;
-    padding: 0 var(--spacing-double);
-  }
+  data () {
+    return {
+      url: '',
```

Comment:
I thought you defined this in `.env` now?

Reply:
https://nuxtjs.org/api/configuration-env/

Reply:
Yes, but this change is a requirement on the Enhanced sharing button, so in order not to mix things up and since it's related to it, I'll fix it when working on that ticket.